source/src/main/java/com/adataptivescale/rosetta/source/core/SourceGeneratorFactory.java

```diff
@@ -1,37 +1,60 @@
 package com.adataptivescale.rosetta.source.core;
 
 import com.adaptivescale.rosetta.common.JDBCDriverProvider;
+import com.adaptivescale.rosetta.common.helpers.ModuleLoader;
 import com.adaptivescale.rosetta.common.models.Database;
 import com.adaptivescale.rosetta.common.DriverManagerDriverProvider;
 import com.adaptivescale.rosetta.common.models.Table;
 import com.adaptivescale.rosetta.common.models.input.Connection;
+import com.adaptivescale.rosetta.common.types.RosettaModuleTypes;
 import com.adataptivescale.rosetta.source.core.interfaces.ColumnExtractor;
 import com.adataptivescale.rosetta.source.core.interfaces.Generator;
+import lombok.extern.slf4j.Slf4j;
 
+import java.lang.reflect.InvocationTargetException;
 import java.util.Collection;
+import java.util.Optional;
 
+@Slf4j
 public class SourceGeneratorFactory {
 
     public static Generator<Database, Connection> sourceGenerator(Connection connection) {
         return sourceGenerator(connection, new DriverManagerDriverProvider());
     }
 
-    public static Generator<Database, Connection> sourceGenerator(Connection connection, JDBCDriverProvider driverProvider) {
-        TablesExtractor tablesExtractor = new TablesExtractor();
+    private static ColumnsExtractor loadColumnExtractor(Connection connection) {
+        Optional<Class<?>> columnExtractorModule = ModuleLoader.loadModuleByAnnotationClassValues(
+                ColumnsExtractor.class.getPackageName(), RosettaModuleTypes.COLUMN_EXTRACTOR, connection.getDbType());
+        if(columnExtractorModule.isEmpty()) {
+            log.warn("Columns extractor not supported for database type: {} falling back to default. ", connection.getDbType());
```

Comment:
This WARN will be printed almost every time, since all the DB Types are using the default COLUMN_EXTRACTOR. Can this be changed to INFO or something else, because if you miss the COLUMN_EXTRACTOR it does not mean that is something wrong.